baseapp-content-feed/baseapp_content_feed/graphql/queries.py

```
@@ -0,0 +1,8 @@
+from baseapp_core.graphql import Node
```

Ruff (0.7.0)

`baseapp_core.graphql.Node` imported but unused

Remove unused import: `baseapp_core.graphql.Node`

(F401)